### DISCUSSION TITLE
Storybook with QuickStart

### DISCUSSION THREAD

**Discussion by mackswell13** [bigcommerce/catalyst#613] [Feature Requests]
I am not sure if this is by design or a Bug but I have done both the quick start, and the manual setup of the catalyst repo, and have noticed an issue on the quick start side. When initializing a quick start repo there seems to be an issue with story book not initializing. It would be nice to be able to use the quick start command line tool and have storybook run right away. This could also be a bug so if that is the case let me know and I can open an issue on the repo!

**Comment by becomevocal** [comment]
@mackswell13 what are you seeing in the terminal when storybook fails to start?

We've actually been talking about whether it's best to run storybook automatically or not, since it's used when you are in component development mode vs. just trying things out (quick start mode). If we made an explicit CLS command to run both storybook and the core app, would that be ok?

**Reply by mackswell13** [reply]
That would be super helpful! I was just wondering more if it was something that should be running. I believe in quick start mode it isn't even trying to run the storybook instance, so I was just making sure that that was by design.
